### PR TITLE
Todo-10-FE: As a user, I want to see all users list

### PR DESCRIPTION
**Pull request by itc02**
image

### CODE REVIEW

**Comment by itc02** [comment]
Why do you reassign PR me?

File: src/components/pagination/styles.tsx [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import styled from 'styled-components';
+
+export const StyledPagination = styled.div`
+  display: flex;
+  justify-content: flex-end;
+`;
```
Comment (Deathlive) [inline src/components/pagination/styles.tsx:6]: `styled-components` Es peny es tiyel sirechivuch li..

File: src/components/dialogs/showUsers/ShowUsersDialog.tsx [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import TableFooter from '@material-ui/core/TableFooter';
+import Paper from '@material-ui/core/Paper';
+import { routes, pagination } from '../../../config/constants';
+import { Transition } from '../addTodo/styles';
+import { UsersData } from '../../../utils/interfaces/users';
+import { TodoCheckbox } from '../../checkbox/TodoCheckbox';
+import { TodoPagination } from '../../pagination/TodoPagination';
```
Comment (Deathlive) [inline src/components/dialogs/showUsers/ShowUsersDialog.tsx:19]: Mojet prosto rename aninq `Pagination` and `Checkbox` qani ver 2-tx ogtagortumnq ?

File: src/components/dialogs/showUsers/ShowUsersDialog.tsx [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+    });
+  }
+
+  const deleteUsers = () => {
+    axios.delete(`${routes.server}/${routes.users}`, { data: {
+      ids: chosenUsers
+    }}).then(res => {
+      setUsers(res.data.users);
+      setTodosNumber(res.data.todos_number);
+      setAllUsersCount(res.data.total_record_count);
```
Comment (Deathlive) [inline src/components/dialogs/showUsers/ShowUsersDialog.tsx:57]: Sta prosto kara ini

`res.data.users.length`

Comment (itc02) [reply src/components/dialogs/showUsers/ShowUsersDialog.tsx:57]: `res.data.users`-y en usernenn, verunq hamapatasxanumn tvyal page-in ev per-in. Isk `res.data.total_record_count`-y sax userneri qanakna. Ozyum asim `res.data.users`-i lengthy karoxa 5 ini (entadrinq min egum amenashaty karoxa 5 todo ini), isk `res.data.total_record_count`-y 15 (yndameny 15 user ka)

Comment (Deathlive) [reply src/components/dialogs/showUsers/ShowUsersDialog.tsx:57]: Lav et qez loxchin county petqa apa sta lav varianta:+1:

File: src/components/checkbox/TodoCheckbox.tsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import React, { useEffect } from 'react';
+import Checkbox from '@material-ui/core/Checkbox';
+
+interface Props {
+  itemId: number;
+  chosenItems: number[];
```
Comment (arpinehp) [inline src/components/checkbox/TodoCheckbox.tsx]: `selectedItems` sounds better I guess

File: src/components/checkbox/TodoCheckbox.tsx [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  setChosenItems: (items: number[]) => void;
```
Comment (arpinehp) [inline src/components/checkbox/TodoCheckbox.tsx]: `setSelectedItems`?

File: src/components/checkbox/TodoCheckbox.tsx [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+  isDisabled?: boolean;
+}
+
+const TodoCheckbox = ({ itemId, chosenItems, setChosenItems, isDisabled }: Props) => {
```
Comment (arpinehp) [inline src/components/checkbox/TodoCheckbox.tsx]: I prefer this syntax `export const TodoCheckbox: React.FC<Props> = ({ itemId, chosenItems, setChosenItems, isDisabled }) => {`

Comment (itc02) [reply src/components/checkbox/TodoCheckbox.tsx]: 👍

File: src/components/checkbox/TodoCheckbox.tsx (hunk 5 above)
Comment (itc02) [reply src/components/checkbox/TodoCheckbox.tsx]: 👍

File: src/components/checkbox/TodoCheckbox.tsx (hunk 4 above)
Comment (itc02) [reply src/components/checkbox/TodoCheckbox.tsx]: 👍

File: src/components/checkbox/TodoCheckbox.tsx (hunk 6 above)
Comment (itc02) [reply src/components/checkbox/TodoCheckbox.tsx]: But I need to `export default` this component as Armen said in the second comment.

Comment (Deathlive) [reply src/components/checkbox/TodoCheckbox.tsx]: User-in Listumy ogtagortums `TodoCheckbox` anunen shpotechnuma.
Prosto poxe **Checkbox** tir, qani ver inqu menak Todo-in chi patkanum.

Ev nuin dzevi ham UserListumy ham TodoListumy poxe arden.
```
import { Checkbox } from '../../checkbox/Checkbox';
import { Pagination } from '../../pagination/Pagination';
```

Comment (itc02) [reply src/components/checkbox/TodoCheckbox.tsx]: **TodoCheckbox** componenty stexcelis ogtvalum `react-material`-i `Checkbox` componentic. Tra heti im komponenti anuny karumchum Checkbox tinim. Arden ti anun goyutyun one. Tra heti et komponenty `export default`-um yral, hancu shtx ozim import anim, kirim:
`import Checkbox from '../../checkbox/Checkbox';`

**Review by Demigoddd — APPROVED** [review]

**Review by arpinehp — APPROVED** [review]

**Pull request merged**